modules/images-update-global-pull-secret.adoc

```diff
@@ -25,18 +25,43 @@ As of {product-title} 4.7, changes to the global pull secret no longer trigger a
 
 .Prerequisites
 
-* You have a new or modified pull secret file to upload.
 * You have access to the cluster as a user with the `cluster-admin` role.
 
+
 .Procedure
+To update the global pull secret, complete the following steps:
+[NOTE]
+====
+To overwrite your existing pull secret using a new pull secret file, skip steps 1 & 2.
+====
+
+. Fetch the existing pull secrets from your pull secret file:
++
+[source,terminal]
+----
+$ oc get secret/pull-secret -n openshift-config --template='{{index .data ".dockerconfigjson" | base64decode}}' ><pull_secret_location> <1>
+----
+<1> Provide the path to the pull secret file.
+
+. Add the new pull secret in your pull secret file:
++
+[source,terminal]
+----
+$ oc registry login --registry="<registry_url>" \ <1>
```

Comment:
That's not URL. Either `<hostname>[:<port>]` or just `<registry>`. Podman uses [[registry]]( http://docs.podman.io/en/latest/markdown/podman-login.1.html).

Reply:
@dmage renamed it to registry